tools/collect-workflow.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+PROG=$(basename "$0")
+
+usage() {
+    echo "$PROG: <-d collection_dir> <-w workflow> [-o yaml| -o json]"
+    echo
+    echo "   -d collection_dir   Directory to create for data collection."
+    echo "   -w workflow         Name of workflow."
```

Comment:
In the future, it might be nice to be able so specify the flux ID rather than workflow.

Reply:
I'll defer this. If the workflow had a label with the jobid then this would be easy. Otherwise, it's just brute-force to walk through all of the workflows to find the jobid.

We could modify create_cb() in coral2_dws.py to do that: https://github.com/flux-framework/flux-coral2/blob/master/src/modules/coral2_dws.py#L133

That could add a label like this to the Workflow resource:
dataworkflowservices.github.io/workflow.jobid=fx6fZheTXG3

Reply:
Couldn't you filter workflows by `.spec.jobID` to retrieve the workflow name and go from there?